ale/formatters/isis_formatter.py

```diff
@@ -0,0 +1,73 @@
+import json
+
+from ale.transformation import FrameNode
+from ale.rotation import ConstantRotation, TimeDependentRotation
+
+def to_isis(driver):
+    meta_data = {}
+
+    meta_data['CameraVersion'] = driver.sensor_model_version
+
+    meta_data['NaifKeywords'] = driver.isis_naif_keywords
+
+    j2000 = driver.frame_chain
+
+    instrument_pointing = {}
+    sensor_frame = j2000.find_child_frame(driver.sensor_frame_id)
+    time_dependent_sensor_frame = j2000.last_time_dependent_frame_between(sensor_frame)
+    if time_dependent_sensor_frame != j2000:
+        forward_path, reverse_path = j2000.path_to(time_dependent_sensor_frame)
+        instrument_pointing['TimeDependentFrames'] = [frame.id for frame in (forward_path + reverse_path)[::-1]]
```

Comment:
The reversal here is to conform to ISIS. It has a slightly strange ordering for these.

Reply:
Can this comment be inlined into the source.